src/guidellm/logger.py

```diff
@@ -64,56 +55,25 @@ def configure_logger(config: Optional[LoggerConfig] = None):
     :type config: LoggerConfig
     """
 
-    _ENV_CONFIG = LoggerConfig(
-        disabled=os.getenv("GUIDELLM_LOG_DISABLED") == "true",
-        clear_loggers=os.getenv("GUIDELLM_CLEAR_LOGGERS") == "true",
-        console_log_level=os.getenv("GUIDELLM_LOG_LEVEL"),
-        log_file=os.getenv("GUIDELLM_LOG_FILE"),
-        log_file_level=os.getenv("GUIDELLM_LOG_FILE_LEVEL"),
-    )
-
-    if not config:
-        config = LoggerConfig()
-    # override from environment variables, if set
-    logger_config = LoggerConfig(
-        disabled=_ENV_CONFIG.disabled or config.disabled,
-        console_log_level=_ENV_CONFIG.console_log_level or config.console_log_level,
-        log_file=_ENV_CONFIG.log_file or config.log_file,
-        log_file_level=_ENV_CONFIG.log_file_level or config.log_file_level,
-    )
-
-    if logger_config.disabled:
+    if config.disabled:
         logger.disable("guidellm")
         return
 
     logger.enable("guidellm")
 
-    if logger_config.clear_loggers:
+    if config.clear_loggers:
         logger.remove()
 
-    if logger_config.console_log_level:
+    if config.console_log_level:
         # log as a human readable string with the time, function, level, and message
         logger.add(
             sys.stdout,
-            level=logger_config.console_log_level.upper(),
+            level=config.console_log_level.upper(),
             format="{time} | {function} | {level} - {message}",
         )
 
-    if logger_config.log_file or logger_config.log_file_level:
-        log_file = logger_config.log_file or "guidellm.log"
-        log_file_level = logger_config.log_file_level or "INFO"
+    if config.log_file or config.log_file_level:
+        log_file = config.log_file or "guidellm.log"
+        log_file_level = config.log_file_level or "INFO"
         # log as json to the file for easier parsing
         logger.add(log_file, level=log_file_level.upper(), serialize=True)
-
-
-# invoke logger setup on import with default values enabling console logging with INFO
-# and disabling file logging
-configure_logger(
```

Comment:
We should keep the root configuration here in the event someone uses the repo as a library rather than going through the main CLI so we always have a base configuration setup and will log out properly. In this way, both CLIs and the library will log the same while enabling users to either override through code or through env variables

Reply:
I don't understand what you mean by the **_root configuration_**.
Do you mean moving the `LoggingSettings` class there?

Reply:
Not quite, I mean that we should invoke configure_logger() in this base logger file like it had before so that on import into the guidellm's __init__.py (and therefore any imports from guidellm) it will trigger the setup of the logger

Reply:
ahh, now I got the idea